Controls/Cafe/Customer1.c

```diff
@@ -19,11 +29,25 @@ void Customer1_checkout(char** name, int* cost) {
  */
 void Task_Customer_1(void* p_arg) {
 	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
-	
+	CPU_TS ticks;
+
 	// Use these values as parameters for Customer1_checkout
 	char* name;
 	int cost;
 	while(1){
-		
+		// Acquire register
+		OSMutexPend(&RegisterOccupied_Mutex, 0, OS_OPT_PEND_BLOCKING, &ticks, &err);
+		if (err)
+			printf("Error code (Cus1 MutexPend): %d\n", err);
```

Comment:
```suggestion
		if (err){
			printf("Error code (Cus1 MutexPend): %d\n", err);
		}
```

According to [our style guide](https://bps.readthedocs.io/en/latest/How_To/coding-style.html), you should have curly brackets around conditional statements, even one-liners. This requirement can be found in the "Punctuation" section under "Miscellaneous".

Reply:
Will put curly braces from now on👀